scripts/init-compliance-tool-for-environment.sh

```diff
@@ -1,57 +1,88 @@
 #!/usr/bin/env bash
 set -e
 
-if [ ! $# -eq 2 ] ; then
-	echo "this script requires the environment and a persistentId to be passed in as arguments. e.g."
-	echo "$ ./init-compliance-tool.sh <environment> <persistentId>"
+if [ ! $# -eq 3 ] ; then
+	echo "For a matching journey this script must include the environment, the --matching flag and a PID"
+	echo "For example: $ ./init-compliance-tool-for-environment.sh <environment> --matching <persistentId>"
+	echo "For a matching journey this script must include the environment, the --nonmatching flag and the filepath to the matching dataset json"
```

Comment:
Shouldn't that say "For a non-matching journey"?

Reply:
Good spot; now corrected.